editor/renames_map_3_to_4.cpp

```diff
@@ -617,52 +623,6 @@ const char *RenamesMap3To4::gdscript_function_renames[][2] = {
 
 // gdscript_function_renames clone with CamelCase
 const char *RenamesMap3To4::csharp_function_renames[][2] = {
```

Comment:
For properties I did a full comparative review of GDScript vs C#, but not for this one.